exemplars/main/rhg.xml

```diff
@@ -1,23 +0,0 @@
-<?xml version="1.0" encoding="UTF-8" ?>
-<!DOCTYPE ldml SYSTEM "../../common/dtd/ldml.dtd">
-<!-- Copyright © 1991-2020 Unicode, Inc.
-For terms of use, see http://www.unicode.org/copyright.html
-Unicode and the Unicode Logo are registered trademarks of Unicode, Inc. in the U.S. and other countries.
-CLDR data files are interpreted according to the LDML specification (http://unicode.org/reports/tr35/)
--->
-<ldml>
-	<identity>
-		<version number="$Revision$"/>
-		<language type="rhg"/>
-	</identity>
-	<layout>
-		<orientation>
-			<characterOrder>right-to-left</characterOrder>
-			<lineOrder>top-to-bottom</lineOrder>
-		</orientation>
-	</layout>
-	<characters>
-		<exemplarCharacters draft="unconfirmed">[\u08EA\u08EB\u08EC\u08ED\u08EE\u08EF \u064B \u08E7 \u064C \u08E8 \u064D \u08E9 \u064E \u08E4 \u064F \u08E5 \u0650 \u08E6 ء ࢬ ا ب پ ت ث ٹ ج چ ح خ د ذ ڈ ر ز ڑ ࢪ س ش ص ض ط ظ ع غ ڠ ف ق ك گ ل م ن ہ و ࢫ ي]</exemplarCharacters>
```

Comment:
should this move to `rhg_Arab.xml`? or maybe `exemplars/main/rhg_Arab.xml` even?

Reply:
Yes, I was thinking about that. Should probably move to exemplars/main/rhg_Arab.xml, I will do that

Reply:
OK, made that change

Reply:
Aargh, test failure from that, looking into it